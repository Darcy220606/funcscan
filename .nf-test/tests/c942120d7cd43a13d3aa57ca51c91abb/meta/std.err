curl: (6) Could not resolve host: www.nextflow.io
WARNING: Could not load nf-core/config profiles: https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config
WARNING: Could not load nf-core/config/funcscan profiles: https://raw.githubusercontent.com/nf-core/configs/master/pipeline/funcscan.config
WARNING: Could not load nf-core/config profiles: https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config
WARNING: Could not load nf-core/config/funcscan profiles: https://raw.githubusercontent.com/nf-core/configs/master/pipeline/funcscan.config
WARNING: Could not load nf-core/config profiles: https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config
WARNING: Could not load nf-core/config/funcscan profiles: https://raw.githubusercontent.com/nf-core/configs/master/pipeline/funcscan.config
[0;31mThe following invalid input values have been detected:

* --input: the file or directory 'https://raw.githubusercontent.com/nf-core/test-datasets/funcscan/samplesheet_reduced.csv' does not exist.
[0m
 -- Check script '/home/james/git/nf-core/funcscan/./workflows/../subworkflows/local/utils_nfcore_funcscan_pipeline/../../nf-core/utils_nfvalidation_plugin/main.nf' at line: 57 or see '/home/james/git/nf-core/funcscan/.nf-test/tests/c942120d7cd43a13d3aa57ca51c91abb/meta/nextflow.log' file for more details
