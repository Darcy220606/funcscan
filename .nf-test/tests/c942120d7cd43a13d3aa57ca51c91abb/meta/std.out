ERROR ~ ERROR: Validation of pipeline parameters failed!

 -- Check '/home/james/git/nf-core/funcscan/.nf-test/tests/c942120d7cd43a13d3aa57ca51c91abb/meta/nextflow.log' file for details
